packages/browser/src/eventbuilder.ts

```diff
@@ -87,6 +87,8 @@ export function eventFromUnknownInput(
 
     event = eventFromString(message, syntheticException, options);
     addExceptionTypeValue(event, message);
+    event.tags = domException.code ? { ...event.tags, 'DOMException.code': String(domException.code) } : event.tags;
```

Comment:
`eventFromString` has no way to add any tags to the event itself, so this fallback will always produce an event with malformed tags:

```js
const event = {}
console.log(event) // {}
event.tags = event.tags
console.log(event) // {tags:undefined}
```

Reply:
Ah, good catch. Thanks.